pulp_rpm/app/shared_utils.py

```diff
@@ -22,8 +24,15 @@ def _prepare_package(artifact, filename):
     # Copy file to a temp directory under the user provided filename
     with tempfile.TemporaryDirectory() as td:
         temp_path = os.path.join(td, filename)
-        shutil.copy2(artifact.file.path, temp_path)
-        cr_pkginfo = createrepo_c.package_from_rpm(temp_path)
+        try:
+            shutil.copy2(artifact.file.path, temp_path)
+            cr_pkginfo = createrepo_c.package_from_rpm(temp_path)
+        except NotImplementedError:
```

Comment:
Is it possible to check either exception code or message?  NotImplementedError is very generic, imo.  We might hide some other issue than just absolute paths with that.

Reply:
I'll do it